__frzr-deploy

```diff
@@ -325,6 +325,40 @@ main() {
 		tar xfO ${IMG_FILE} | btrfs receive --quiet ${DEPLOY_PATH}
 	fi
 
+	if [ -d "${SUBVOL}/etc" ]; then
+		echo "System image ${NAME} will use the /etc overlay"
+	else
+		echo "System image ${NAME} will use an /etc subvolume"
+
+		# unlock the subvolume to be able to create a nested subvolume
+		btrfs property set -f -ts "${SUBVOL}/" ro false
+
+		# create the nested subvolume for /etc
+		btrfs subvolume create "${SUBVOL}/etc"
+
+		# create the nested subvolume for /snapshots
+		btrfs subvolume create "${SUBVOL}/snapshots"
+
+		# re-lock the subvolume. ${SUBVOL}/etc will remain R/W
+		btrfs property set -f -ts "${SUBVOL}/" ro true
+
+		btrfs subvolume create "${SUBVOL}/snapshots/etc"
+
+		# copy every stock /etc (that has been moved to /usr/etc) file to the actual /etc
+		cp -a ${SUBVOL}/usr/etc/* ${SUBVOL}/etc/
+
+		# copy the machine-id file: this was created by systemd the very first boot and identify the machine:
+		# changing this will also make ssh warn about machine not matching.
+		if [ -f "/etc/machine-id" ]; then
```

Comment:
Is this necessary? Do we have a mchine-id in our system image such that this would get overwritten? If we do, we should remove it.

Reply:
This PR, as the hibernation one, was superceeded by the upcoming frzr rework. Machine-id file is created at first boot by systemd on the first boot. I don't know if we have one in our image, but for sure it is something to investigate.

However I don't think we have one simply because systemd is not run from arch-chroot.